### PR TITLE
Changing to Maria db documentation updated

### PR DESCRIPTION
**Pull request by Buddhimah**

### CODE REVIEW

File: en/docs/administer/changing-to-mariadb.md [hunk 1]
```diff
@@ -1,85 +1,164 @@
...
+                     data.
+- `WSO2_IDENTITY_DB` - The database specific for the identity server which stores
+                       identity related data
+                       
+After setting up MariaDB database. You can point the `WSO2_IDENTITY_DB` or 
+`WSO2_SHARED_DB` or both to that MariaDB database by following below instructions.
+
+!!! note
+       
+    WSO2 Identity Server currently does not provide specific MariaDB SQL scripts.However
```
Comment (gomathyk) [inline en/docs/administer/changing-to-mariadb.md]: does not provide scripts specific to MariaDB SQL.

File: en/docs/administer/changing-to-mariadb.md [hunk 2]
```diff
@@ -1,85 +1,164 @@
...
+            minIdle = "5"
+            testOnBorrow = true
+            validationQuery="SELECT 1"
+            validationInterval="30000"
+            defaultAutoCommit=false
+           ```
+           
+       1. Executing database scripts.
+        
+          Navigate to `<IS-HOME>/dbscripts`. Execute the scripts in the following file, against the database created.
```
Comment (gomathyk) [inline en/docs/administer/changing-to-mariadb.md]: Remove comma

File: en/docs/administer/changing-to-mariadb.md [hunk 3]
```diff
@@ -1,85 +1,164 @@
...
+                      
+           - `<IS-HOME>/dbscripts/mysql.sql`
+           
+   3. If you have a requirement in using workflow feature follow,
```
Comment (gomathyk) [inline en/docs/administer/changing-to-mariadb.md]: If you require the workflow feature, follow

File: en/docs/administer/changing-to-mariadb.md [hunk 4]
```diff
@@ -1,85 +1,164 @@
...
+       1. Executing database scripts.
+        
+          Navigate to `<IS-HOME>/dbscripts`. Execute the scripts in the following file, against the database created.
+                      
+           - `<IS-HOME>/dbscripts/mysql.sql`
+           
+   3. If you have a requirement in using workflow feature follow, 
+       [Changing the default database of BPS database](../../administer/changing-datasource-bpsds)
+       
+   4.  Download the MariaDB JDBC driver for the version, you are using and
```
Comment (gomathyk) [inline en/docs/administer/changing-to-mariadb.md]: remove comma

File: en/docs/administer/changing-to-mariadb.md [hunk 5]
```diff
@@ -91,181 +170,31 @@ Follow the steps below to change the type of the default datasource.
     </tr>
     <tr class="odd">
     <td><p><strong>testOnBorrow</strong></p></td>
-    <td>The indication of whether objects will be validated before being borrowed from the pool. If the object fails to validate, it will be dropped from the pool, and another attempt will be made to borrow another.</td>
+    <td>Indicates Whether objects will be validated before being borrowed from the pool. If the object fails to
```
Comment (gomathyk) [inline en/docs/administer/changing-to-mariadb.md]: Please change the W in Whether to simple case.

File: en/docs/administer/changing-to-mariadb.md [hunk 6]
```diff
@@ -91,181 +170,31 @@ Follow the steps below to change the type of the default datasource.
...
     <td><p><strong>testOnBorrow</strong></p></td>
-    <td>The indication of whether objects will be validated before being borrowed from the pool. If the object fails to validate, it will be dropped from the pool, and another attempt will be made to borrow another.</td>
+    <td>Indicates Whether objects will be validated before being borrowed from the pool. If the object fails to 
+    validate, it will be dropped from the pool, and another attempt will be made to borrow another.</td>
     </tr>
     <tr class="even">
-    <td><strong>validationQuery</strong></td>
-    <td>The SQL query that will be used to validate connections from this pool before returning them to the caller.</td>
+    <td><p><strong>defaultAutoCommit</strong></p></td>
+    <td>Indicates Whether to commit database changes automatically or not.</td>
```
Comment (gomathyk) [inline en/docs/administer/changing-to-mariadb.md]: Please change the W in Whether to simple case.

File: en/docs/administer/changing-to-mariadb.md (hunk 1 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-mariadb.md]: done

File: en/docs/administer/changing-to-mariadb.md (hunk 2 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-mariadb.md]: done

File: en/docs/administer/changing-to-mariadb.md (hunk 3 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-mariadb.md]: done

File: en/docs/administer/changing-to-mariadb.md (hunk 4 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-mariadb.md]: done

File: en/docs/administer/changing-to-mariadb.md (hunk 5 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-mariadb.md]: done

File: en/docs/administer/changing-to-mariadb.md (hunk 6 above)
Comment (Buddhimah) [reply en/docs/administer/changing-to-mariadb.md]: done

**Pull request merged**